dev/fattest.simplicity/src/componenttest/rules/repeater/EE7FeatureReplacementAction.java

```diff
@@ -20,7 +20,7 @@
 
     static final String[] EE7_FEATURES_ARRAY = { "javaee-7.0", "webProfile-7.0", "javaeeClient-7.0", "servlet-3.1", "jdbc-4.1", "javaMail-1.5", "cdi-1.2", "jpa-2.1",
                                                  "beanValidation-1.1", "jaxrs-2.0", "jaxrsClient-2.0", "jsf-2.2", "appSecurity-2.0", "jsonp-1.0" };
-    static final Set<String> EE7_FEATURE_SET = new HashSet<>(Arrays.asList(EE7_FEATURES_ARRAY));
+    public static final Set<String> EE7_FEATURE_SET = new HashSet<>(Arrays.asList(EE7_FEATURES_ARRAY));
```

Comment:
We should wrap this in `Collections.unmodifiableSet` so that the value cannot be accidentally changed by other FATs

Reply:
applied the change to a new commit, please review.